src/gateways/GovNotify/index.test.js

```diff
@@ -10,4 +15,36 @@ describe("GovNotify", () => {
 
     expect(instanceOne).toEqual(instanceTwo);
   });
+
+  describe("when using production environment", () => {
+    beforeEach(() => {
+      process.env.NODE_ENV = "production";
```

Comment:
Since the code only looks at the APP_ENV, do we need to be changing the NODE_ENV here as it has no affect on the fake client logic? In production the APP_ENV is simply not defined

Reply:
Yeah, I went back and forth with this. Will remove!